README.md

```diff
@@ -108,6 +112,119 @@ async onRequestInit({ requestInit }) {
 }
 ```
 
+If you're accepting webhooks from Clerk, you will need to validate the incoming webhook signature. This can be done like this:
+
+```cs
+using Newtonsoft.Json;
+using System.IO;
+using System.Threading.Tasks;
+using System.Collections.Generic;
+using Svix;
+using Svix.Models;
+using System.Net;
+
+[ApiController]
+[Route("api/[controller]")]
+public class ClerkController : ControllerBase
+{
+    private readonly IConfiguration _configuration;
+
+    public ClerkController(IConfiguration configuration)
+    {
+        _configuration = configuration;
+    }
+
+    [HttpPost("webhook")]
+    public async Task<IActionResult> ClerkWebhook()
+    {
+        // Retrieve the Clerk webhook secret from the configuration
+        var clerkWebhookSecret = "your-webhook-secret-here";
+
+        // Read the request body
+        var json = await new StreamReader(HttpContext.Request.Body).ReadToEndAsync();
+        Console.WriteLine($"Payload: {json}");
+
+        // Retrieve the Svix headers
+        var svixId = Request.Headers["svix-id"].ToString();
+        var svixTimestamp = Request.Headers["svix-timestamp"].ToString();
+        var svixSignature = Request.Headers["svix-signature"].ToString();
+
+        Console.WriteLine($"Svix ID: {svixId}");
+        Console.WriteLine($"Svix Timestamp: {svixTimestamp}");
+        Console.WriteLine($"Svix Signature: {svixSignature}");
+
+        if (string.IsNullOrEmpty(svixId) || string.IsNullOrEmpty(svixTimestamp) || string.IsNullOrEmpty(svixSignature))
+        {
+            return BadRequest("Missing headers");
+        }
+
+        // Attempt signature verification with the raw signature
+        var wh = new Webhook(clerkWebhookSecret);
+        WebHeaderCollection headers = new WebHeaderCollection
+        {
+            { "svix-id", svixId },
+            { "svix-timestamp", svixTimestamp },
+            { "svix-signature", svixSignature }
+        };
+
+        Event webhookEvent;
+        try
+        {
+            wh.Verify(json, headers); // Verify doesn't return an event, just verifies the signature
+            webhookEvent = JsonConvert.DeserializeObject<Event>(json); // Deserialize the JSON into an Event object
```

Comment:
Replace Newtonsoft usage with STJ. Newtonsoft shouldn't be used for modern examples.

Reply:
Good point